samples/guide/src/main/java/okhttp3/recipes/CancelCall.java

```diff
@@ -24,6 +24,7 @@
 import okhttp3.Request;
 import okhttp3.Response;
 
+@SuppressWarnings("FutureReturnValueIgnored")
```

Comment:
remove these from sample code? Maybe we can suppress the warnings in the build file instead of the source?

Reply:
It grosses me out that our users will just copy these suppressions into their code, not knowing why they exist or what they do